models/system/distilbert-base-uncased-distilled-squad/spec.yaml

```diff
@@ -6,6 +6,12 @@ properties:
   datasets: squad
   finetuning-tasks: text-classification, token-classification, question-answering
   languages: en
+  inference-recommended-sku: Standard_DS2_v2
```

Comment:
is there a requirement of `min_inference_sku` after providing `inference-recommended-sku`?

Reply:
There shouldn't be, can we remove it? @haritha-bendapudi @corymcc @jpmann

Reply:
Updated